eng/restore/netcoreapp/netcoreapp.depproj

```diff
@@ -4,15 +4,23 @@
   <PropertyGroup>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
     <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.7</NETCoreAppPackageVersion>
-    <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">2.1.3</NETCoreAppPackageVersion>
+    <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">2.1.3</NETCoreAppPackageVersion>        
+    <RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp3.0'">3.0.0-preview7-27912-14</RuntimeFrameworkVersion>
```

Comment:
We need a comment here with an issue to replace this with RTM when available.  Even better would be to get this from packageIndex.  /cc @joperezr